src/sentry/static/sentry/app/components/commitAuthorStats.jsx

```diff
@@ -113,35 +104,32 @@ const CommitAuthorStats = createReactClass({
 
     return (
       <div style={{marginTop: 5}}>
-        <h6 className="nav-header m-b-1">Commits by Author</h6>
+        <h6 className="nav-header m-b-1">{t('Commits by Author')}</h6>
         {!commitAuthorValues.length && this.renderEmpty()}
-        <ul className="list-group">
-          {commitAuthorValues.map((commitAuthor, i) => {
-            let {author, commitCount} = commitAuthor;
-            return (
-              <li
-                key={i}
-                className="list-group-item list-group-item-sm list-group-avatar"
-              >
-                <div className="row row-flex row-center-vertically">
-                  <div className="col-sm-8">
-                    <span
-                      className="avatar-grid-item m-b-0 tip"
-                      title={author.name + ' ' + author.email}
-                    >
+        <Panel>
+          <PanelBody>
+            {commitAuthorValues.map((commitAuthor, i) => {
+              let {author, commitCount} = commitAuthor;
+              return (
+                <PanelItem key={i} p={1} align="center">
+                  <Flex>
+                    <Tooltip title={author.name + ' ' + author.email}>
                       <Avatar user={author} size={20} />
-                    </span>
+                    </Tooltip>
+                  </Flex>
+                  <Flex style={{flexGrow: 1}} px={1}>
                     <CommitBar
+                      style={{marginLeft: 5}}
                       totalCommits={commitList.length}
                       authorCommits={commitCount}
                     />
-                  </div>
-                  <div className="col-sm-4 align-right">{commitCount}</div>
-                </div>
-              </li>
-            );
-          })}
-        </ul>
+                  </Flex>
+                  <Flex>{commitCount}</Flex>
```

Comment:
Does this need to be flex?

We may want this and the avatar container to have `flex-shrink: 0`, since CommitBar has `flex-grow`